site/src/components/pages/dashboard/DashboardCard.tsx

```diff
@@ -1,31 +1,166 @@
+  onClick?: () => void;
+  href?: string;
+};
+
+const CardWrapper: FC<CardWrapperProps> = ({ children, onClick, href }) => {
+  const sharedStyles: BoxProps["sx"] = useMemo(
+    () => ({
+      position: "relative",
+
+      "&::after": {
+        content: `""`,
+        position: "absolute",
+        top: 0,
+        left: 0,
+        height: "100%",
+        width: "100%",
+        boxShadow: 1,
+        opacity: 0,
+        transition: ({ transitions }) =>
+          transitions.create("opacity", { duration: 300 }),
+      },
+
+      "&:hover::after": {
+        opacity: 1,
+      },
+
+      "&:focus-visible": {
+        outline: ({ palette }) => `1px solid ${palette.purple[700]}`,
+      },
+    }),
+    [],
+  );
+
+  if (onClick) {
+    return (
+      <Box
+        component="button"
+        sx={{ textAlign: "left", ...sharedStyles }}
+        onClick={onClick}
+      >
+        {children}
+      </Box>
+    );
+  }
+
+  return (
+    <Link href={href} sx={sharedStyles}>
+      {children}
+    </Link>
+  );
+};
+
+export const DashboardCardSecondary: VoidFunctionComponent<
+  Pick<DashboardCardProps, "title" | "link" | "description" | "icon">
+> = ({ link, title, description, icon }) => {
+  return (
+    <CardWrapper href={link.href} onClick={link.onClick}>
+      <Box
+        sx={{
+          display: "flex",
+          border: ({ palette }) => `1px solid ${palette.gray[30]}`,
+          borderRadius: "4px",
+          p: 3,
+        }}
+      >
+        <Icon
+          sx={{ fontSize: 32, mr: 3, color: ({ palette }) => palette.gray[40] }}
+          className={icon}
+        />
+        <Box>
+          <Typography variant="bpLargeText" sx={{ mb: 0.75 }}>
+            {title}
+          </Typography>
+          <Typography sx={{ mb: 2 }}>{description}</Typography>
+          <Box
+            sx={{
+              color: ({ palette }) => palette.purple[700],
+              fontWeight: 600,
+              path: {
+                fill: "currentColor",
+              },
+              display: "flex",
+              alignItems: "center",
+            }}
+          >
+            <Box component="span" paddingRight={1}>
+              {link.title}
+            </Box>
+            <ArrowRightIcon
+              sx={{
+                width: "auto",
+                height: "0.8em",
+              }}
+            />
+          </Box>
+        </Box>
+      </Box>
+    </CardWrapper>
+  );
 };
 
 export const DashboardCard: VoidFunctionComponent<DashboardCardProps> = ({
   title,
   colorGradient,
   description,
   link,
+  variant,
+  icon,
 }) => {
+  if (variant === "secondary") {
+    return (
+      <DashboardCardSecondary
+        {...{ title, description, link, variant, icon }}
+      />
+    );
+  }
+
   return (
-    <Link href={link.href} sx={{ marginBottom: 4 }}>
+    <CardWrapper href={link.href} onClick={link.onClick}>
       <Box
         sx={{
+          // @todo update theme config to include microShadow
```

Comment:
Is there a task for this, or a reason it isn't added as a part of this PR?

Reply:
I will create a task for that. The reason it isn't added here is because updating the boxShadows in our theme config will also mean updating all places we use boxShadows in the project. This is because the shadows are array index based
```
// theme config
boxShadows: [none, "<baseShadow>", "<smallShadow>"]

// usage
<Box sx={{ boxShadow: 1 // baseShadow }} />
```
Adding a new shadow, will result in updating usages of boxShadow since they most likely now have a new index

```
boxShadows: [none, "microShadow", "<baseShadow>", "<smallShadow>"]
// New usage
<Box sx={{ boxShadow: 2 // baseShadow }} />
```